api/src/opentrons/protocol_engine/state/substore.py

```diff
@@ -2,13 +2,12 @@
 from abc import ABC, abstractmethod
 from typing import Generic, TypeVar
 
-from ..commands import Command
-
+from .actions import Action
 
 SubstateT = TypeVar("SubstateT")
 
 
-class Substore(ABC, Generic[SubstateT]):
+class HasState(ABC, Generic[SubstateT]):
     """Abstract base class for a sub-store."""
```

Comment:
```suggestion
    """Abstract base class for something that holds substate."""
```

Maybe, for the sake of consolidating terminology?

(Otherwise, what is a "substore", and why would the class documented as representing a substore not be called `Substore`?)

Does the module name `substore` still make sense?

Reply:
> Does the module name substore still make sense?

No it does not, especially once I realized this interfaces (`HasState` and `HandlesAction`) also apply to the top-level `StateStore`. Will rename and fixup docstrings accordingly